Makefile

```diff
@@ -108,194 +108,268 @@ BASE_LDFLAGS = ${SHRINKFLAGS} \
 
 GO_LDFLAGS = -ldflags '${BASE_LDFLAGS} ${EXTRA_LDFLAGS}'
 
+define go-build
+	$(shell cd `pwd` && $(GO_BUILD) -o $(BUILD_BIN_PATH)/$(shell basename $(1)) $(1))
+	@echo > /dev/null
+endef
+
+define curl_to
+    curl -sSfL --retry 5 --retry-delay 3 "$(1)" -o $(2)
```

Comment:
A tab missing? (not sure 100% from the diff in GitHub)

Reply:
Fixed :+1: